cdb2api/cdb2api.c

```diff
@@ -1257,6 +1257,11 @@ static int try_ssl(cdb2_hndl_tp *hndl, SBUF2 *sb, int indx)
     cdb2_ssl_sess_list *store;
     SSL_SESSION *sess;
 
+    if (hndl->debug_trace) {
```

Comment:
I am not sure if we should add debug code here: It offers very limited information, and makes the code more difficult to read.